hedvig-lint/lint-baseline/lint-baseline-forever-ui.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.3.1" type="baseline" client="gradle" dependencies="false" name="AGP (8.3.1)" variant="all" version="8.3.1">
+
+    <issue
+        id="UnknownIssueId"
+        message="Unknown issue id &quot;UnsafeOptInUsageError&quot;">
+        <location
+            file="build.gradle.kts"/>
+    </issue>
+
+    <issue
+        id="UnknownIssueId"
+        message="Unknown issue id &quot;UnsafeOptInUsageError&quot;">
+        <location
+            file="build.gradle.kts"/>
+    </issue>
```

Comment:
Interesting that these are here in the first place. Usually these are empty, especially for new modules. If you delete this baseline file and run `./gradlew lint` again does it tell you where these are originating from?

Reply:
empty now:)